rapids_dask_dependency/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) 2024, NVIDIA CORPORATION.
 
-from .dask_loader import DaskLoader
+from .dask_loader import DaskFinder
 
-DaskLoader.install()
+DaskFinder.install()
```

Comment:
**Non-blocking**: Do you think it makes sense to toggle patching with an environment variable, so that we can quickly check if the custom Finder/Loader is causing a hypothetical bug/error in the future?